firefox-ios/Client/Frontend/Browser/BrowserViewController/Views/BrowserViewController.swift

```diff
@@ -17,6 +17,7 @@ import WebEngine
 import WidgetKit
 import SummarizeKit
 import ActivityKit
+import Glean
```

Comment:
Why do we need this import all of a sudden? 🤔

Reply:
After removing a bunch of code for the legacy cleanup from `BrowserViewController+URLBarDelegate.swift` there was little code left in that file, so I moved the remainder to BVC and part of that was a call to `GleanMetrics`.